app/models/theme.rb

```diff
@@ -99,28 +94,28 @@ def self.expire_site_cache!
   end
 
   def self.clear_default!
-    SiteSetting.default_theme_key = ""
+    SiteSetting.default_theme_id = -1
```

Comment:
hmmm is the default 0? cause later on you are checking for blank?

Reply:
I made the default for this setting (i.e. when there is no default theme) -1.

I made sure `theme_key` becomes `nil` when the setting value is -1 so that we don't end up calling `blank?` on a number. See:
https://github.com/discourse/discourse/pull/6073/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR353

Reply:
I'm happy to change it to 0 if it makes more sense.